py_zipkin/zipkin.py

```diff
@@ -109,6 +109,7 @@ def __init__(
         add_logging_annotation=False,
         report_root_timestamp=False,
         use_128bit_trace_id=False,
+        host=None
```

Comment:
add a docstring explaining that this will expect the ipv4 value of the host and that the ipv4 value won't be automatically determined